### ISSUE TITLE
New FIL funding round for Spark

### ISSUE THREAD

**Issue by bajtos** [space-meridian/roadmap#174]
Spark's FIL pot will be depleted soon.

- I topped the IE contract balance with 50 FIL today; its new balance is ~130 FIL. This will last for ~26 days. The next alert will be triggered in ~6 days.
- spark-publish seems to pay less than 0.2 FIL per week and has 71 FIL in the wallet - all is good.
- spark-evaluate spends 5 FIL per week and has 24 FIL left in the wallet - we will need to top it up in 2-4 weeks.
- We have only 25 FIL left in the Spark multisig.

**In other words, we have around 5 weeks before Spark runs out of funds.**
